src/win32.cc

```diff
@@ -81,7 +70,10 @@ bool _InitInstance()
     }
 
     if (!result) {
```

Comment:
[GetVersionExA may be altered or unavailable for releases after Windows 8.1](https://docs.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-getversionexa)

Reply:
We could use `IsWindowsXPOrGreater()` but I think there is no point to do that